test/file-test.js

```diff
@@ -383,3 +381,70 @@ describe('expect(file(...))', function() {
     });
   });
 });
+
+describe('expect(string)', function() {
+  ['.to.equal', '.equals', '.to.eq'].forEach(function(methodDesc) {
+    var method = methodDesc.replace('.to', '').replace('.', '');
+
+    describe(methodDesc + '(file(...))', function() {
+      it('passes for string to equal file', function() {
+        expect('This is a small fixture file called "foo.txt"!\n').to[method](file('test/fixtures/foo.txt'));
+      });
+
+      it('fails for string not equal to file', function() {
+        expect(function() {
+          expect('large solid object').to[method](file('test/fixtures/foo.txt'));
+        }).to.throw(function(err) {
+          expect(err.toString()).to.equal('AssertionError: expected \"test/fixtures/foo.txt\" to equal \"large solid object\"');
```

Comment:
The message is more or less reversed here. Maybe we could add an `inverse` parameter to the `assertEqual()` method to flip it?